dashboard/app/views/pd/application/teacher1920_application_mailer/declined.html.haml

```diff
@@ -3,41 +3,18 @@
   = @application.first_name + ","
 
 %p
-  Thank you for applying to Code.org's Professional Learning Program for
+  Thank you for applying to
+  = @application.regional_partner.name + "'s'"
+  2019-20 Professional Learning Program for
   = @application.course_name + "."
-  We appreciated the opportunity to read your application and learn more about
-  your interest in teaching computer science.
-
-- if @application.regional_partner_id
-  %p
-    We're unfortunately unable to offer you a space this year. Due to record
-    interest in this year's program and limited space, we don't have the capacity
-    to accept all applicants.
-- else
-  %p
-    Our current program model prioritizes candidates in areas where Code.org has
-    established Regional Partners who provide continued learning support
-    throughout the year. Unfortunately, without a local Partner to facilitate your
-    professional development, we cannot offer you a space in the full Professional
-    Learning Program experience this year.
+  Our team appreciated the opportunity to review your application and learn about your
+  commitment to computer science education. I'm writing to let you know we are currently
```

Comment:
tiny: I think this sentence sounds better with the word 'that': "I'm writing to let you know *that* we are currently..."